src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/TokenizationTests.cs

```diff
@@ -15,7 +16,7 @@
 using Microsoft.PowerFx.Syntax;
 using Xunit;
 
-namespace Microsoft.PowerFx.Core.Tests.IntellisenseTests
```

Comment:
>Microsoft.PowerFx.Core.Tests.IntellisenseTests [](http://example.com/codeflow?start=10&length=46)

why namespace change?

Reply:
To be consistent with other tests in the same project. The previous namespace was not matching the tests around the ones in this file. The previous namespace was auto generated by the VS and i didn't notice that it was bit different than the other tests.

Reply:
Feel free to reopen this if you want me to change it back